Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -1162,9 +1172,94 @@ static AffineTransform documentSpaceToPageSpaceTransform(const IntDegrees& pageR
     return false;
 }
 
+#pragma mark Selections
+
+auto UnifiedPDFPlugin::selectionGranularityForMouseEvent(const WebMouseEvent& event) const -> SelectionGranularity
+{
+    if (event.clickCount() == 2)
+        return SelectionGranularity::Word;
+    if (event.clickCount() == 3)
```

Comment:
Perhaps some enums for these?

Reply:
Hmm, I was planning to clean up this code a little more as part of https://webkit.org/b/268616. I'll hold off till then.